SokolTextGame/World.cs

```diff
@@ -11,11 +11,13 @@ private void CommandDictionary()
             commands["who"] = new WhoAmI();
             commands["look"] = new Look();
             commands["where"] = new WhereAmI();
```

Comment:
Please change the name of the `WhereAmI` command to `Where`. In this case, you're processing `where am i` and `where can i go`, with some other inputs possible in the future. This is not just a `WhereAmI` command.